app/assets/stylesheets/sign.css

```diff
@@ -0,0 +1,51 @@
+
+h2 {
+  text-align: center;
+}
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  margin: 2rem auto;
+  align-items: center;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  gap: 0.5rem;
+  align-content: center;
+  border-radius: 10px;
+  padding: 1rem;
+}
+
+input,
+.action {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  height: 2rem;
+  padding-left: 5px;
+  border-radius: 5px;
+  border: none;
+}
```

Comment:
- [ ] [OPTIONAL] Good job aligning items, however, I'll suggest making all `input` accessible to facilitate a user to see them easily by removing the `border: none` in your style 👍 

> | | without `border: none` | | with `border: none` | |
> |-|---------|-|--------|-|
> | |![Screenshot 2023-10-06 140546](https://github.com/tjay1760/Blog-App/assets/57408419/cfe63cf4-358c-45a6-a104-3194fe0b2bb2)| |![Screenshot 2023-10-06 140638](https://github.com/tjay1760/Blog-App/assets/57408419/fba4aed9-c706-4862-a452-c1547a92c1bf)| |